checkpoint3/Program.cs

```diff
@@ -0,0 +1,199 @@
+﻿using System;
```

Comment:
The assignment has the correct functionality, but it is not structured correctly.